apm-sniffer/apm-sdk-plugin/dubbo-2.7.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/asf/dubbo/DubboInterceptor.java

```diff
@@ -110,14 +109,6 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
     public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allArguments, Class<?>[] argumentsTypes,
                               Object ret) throws Throwable {
         Result result = (Result) ret;
-        try {
-            if (result != null && result.getException() != null) {
-                dealException(result.getException());
-            }
```

Comment:
Do we need to process `result#getExcetpion`?

Reply:
I haven't received notifications these days, I will debug and make it clear about how it process Exceptions.

Reply:
Usually, we don't need to catch exceptions, as the `handleMethodException` method exists already. But we should process `result#getExcetpion`, because this exception may throw after the whole intercepting is finished or even wouldn't, depending on Dubbo's implementation.